samples/server/petstore/python-flask/openapi_server/test/test_spec_parsing.py

```diff
@@ -0,0 +1,14 @@
+# coding: utf-8
+
+import connexion
+import logging
+from openapi_server.encoder import JSONEncoder
+from unittest import TestCase
+
+class TestSpecParsing(TestCase):
+    def test(self):
+        logging.getLogger('connexion.operation').setLevel('ERROR')
+        app = connexion.App(__name__, specification_dir='../openapi/')
+        app.app.json_encoder = JSONEncoder
+        app.add_api('broken_openapi.yaml', pythonic_params=True)
+        self.assertTrue(app.app is not None)
```

Comment:
Can you also assert that the datatype of the parameter that you tweaked is correct?